packages/fxa-payments-server/server/lib/server.js

```diff
@@ -22,6 +23,21 @@ module.exports = () => {
 
   const app = express();
 
+  // Each of these config values (e.g., 'servers.content') will be exposed as the given
+  // variable to the client/browser (via fxa-content-server/config)
+  const CLIENT_CONFIG_MAP = {
+    contentUrl: 'servers.content',
+    oAuthUrl: 'servers.oauth',
+    profileUrl: 'servers.profile',
+  };
```

Comment:
I copied the names from https://github.com/mozilla/fxa/blob/master/packages/fxa-content-server/server/lib/routes/get-index.js#L38-L55 – but probably it would be more sensible to just create a partial copy of the object in a naive way, like:

```js
const CLIENT_CONFIG = {
  servers: {
    content: config.get('servers.content'),
    oauth: config.get('servers.oauth'),
    profile: config.get('servers.profile'),
  }
};
```

Reply:
I might do this in #1280